compiler/natives/src/os/signal/signal.go

```diff
@@ -0,0 +1,13 @@
+// +build js
+
+package signal
+
+// Package signal is not implemented for GOARCH=js.
+// TODO: Implement if possible.
```

Comment:
I don't see any reason why we would need to implement it. We don't have signals in the browser.